recipe/meta.yaml

```
@@ -62,11 +68,13 @@ requirements:
     # OpenCL ICD
     - ocl-icd  # [linux]
     - khronos-opencl-icd-loader  # [win or (osx and opencl_impl == 'khronos')]
-    - libcufft-dev  # [(linux64 or ppc64le or win) and (cuda_compiler_version or "").startswith("12")]
-    - cuda-profiler-api  # [(linux64 or ppc64le or win) and (cuda_compiler_version or "").startswith("12")]
-    - cuda-nvrtc-dev  # [(linux64 or ppc64le or win) and (cuda_compiler_version or "").startswith("12")]
-    - cuda-driver-dev  # [(linux64 or ppc64le or win) and (cuda_compiler_version or "").startswith("12")]
-
+    {% if cuda_major >= 12 %}
+    - libcufft-dev  # [linux64 or ppc64le or win]
+    - cuda-profiler-api  # [linux64 or ppc64le or win]
+    - cuda-nvrtc-dev  # [linux64 or ppc64le or win]
```

Would suggest just using `linux` here

```suggestion
    - libcufft-dev  # [linux or win]
    - cuda-profiler-api  # [linux or win]
    - cuda-nvrtc-dev  # [linux or win]
```

---

This breaks aarch64 builds.
```shell
[ 90%] Built target OpenMMPME
[ 90%] Building CXX object plugins/cudacompiler/CMakeFiles/OpenMMCudaCompiler.dir/src/CudaCompilerKernelFactory.cpp.o
In file included from /home/conda/feedstock_root/build_artifacts/openmm_1700463049050/work/platforms/cuda/include/CudaKernels.h:31,
                 from /home/conda/feedstock_root/build_artifacts/openmm_1700463049050/work/plugins/cudacompiler/src/CudaCompilerKernels.h:36,
                 from /home/conda/feedstock_root/build_artifacts/openmm_1700463049050/work/plugins/cudacompiler/src/CudaCompilerKernelFactory.cpp:28:
/home/conda/feedstock_root/build_artifacts/openmm_1700463049050/work/platforms/cuda/include/CudaArray.h:33:10: fatal error: cuda.h: No such file or directory
   33 | #include <cuda.h>
      |          ^~~~~~~~
```
Weird because CUDA packages seem to be there:
```shell
    cuda-driver-dev:                  12.0.107-hac28a21_6            conda-forge
    cuda-driver-dev_linux-aarch64:    12.0.107-hac28a21_6            conda-forge
```

---

Yes because the same change is needed [above]( https://github.com/conda-forge/openmm-feedstock/pull/119/commits/d9c2d289aea6e6550eac299198e21d8e16e2a7b7#diff-f3725a55bf339595bf865fec73bda8ac99f283b0810c205442021f29c06eea9aR47 )